docs/site/Interceptor-generator.md

```diff
@@ -121,8 +121,10 @@ import {
  * This class will be bound to the application as a global `Interceptor` during
  * `boot`
  */
-@bind({tags: {namespace: 'interceptors', name: 'test'}})
+@bind({tags: {key: TestInterceptor.BINDING_KEY}})
```

Comment:
Discussion point: 
- Before my change, we would scaffold the interceptor in such way that it's bound to `interceptors.test` key. 
- With my change in place, it will be bound to `interceptors.TestInterceptor` key.

My reasoning: the binding name should match the class name. When the class is renamed by the developer, the binding name should be updated accordingly.

Thoughts?

/cc @strongloop/loopback-next